src/schemas/header_schema.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel, Field
+
+
+class HeaderSchema(BaseModel):
+    """
+    The Header Scheme describe the header schema for a request.
+    """
+
+    miauserid: str = 'miauserid'
```

Comment:
Is it correct that these values are hardcoded?

Reply:
Are harcoded because otherwise we would have had to develop a nontrivial validation system